src/evm_context.rs

```diff
@@ -116,6 +123,17 @@ pub fn basefee(ssa: &mut SSATracker) -> String {
     ssa.to_smt_variable(&static_vars()["basefee"]).name
 }
 
+pub fn calldataload(index: &String, ssa: &mut SSATracker) -> String {
+    let calldata_var = calldata().name;
+    let arguments = (0..32).map(|i| {
+        format!(
+            "({calldata_var} (bvadd {} #x{i:064X}))",
```

Comment:
Can you take the variables out of the string? Took me forever to see that `i` in `i:064X` was actually referring to the range values